### DISCUSSION TITLE
Add support for links

### DISCUSSION THREAD

**Discussion by ArnaudMarieGallardoTFS** [scalar/scalar#632] [Ideas]
Since OpenAPI 3.0, **links** can be added to show how values returned by one operation can be used as input for other operations. It is a very useful feature to understand how to interact with the API.

Documentation: https://swagger.io/docs/specification/links/

Example:

![CaptureSwaggerLink](https://github.com/scalar/scalar/assets/78972871/d043a7b9-ff3d-475f-a920-b62ad89f242d)

I would be happy to work on a PR if needed!

**Comment by marclave** [comment]
@ArnaudMarieGallardoTFS this is a great idea! Feel free to get started on this ✨
